packages/safety-suite-api/src/api/models/types.ts

```diff
@@ -115,23 +115,281 @@ export type OrderBy = {
 };
 
 export type ModelListOptions = {
+  /**
+   * Maximum number of models to return.
+   */
   limit?: number;
+  /**
+   * Options for sorting the returned models.
+   */
   orderBy?: OrderBy;
+  /**
+   * Index in the array of returned models.
+   */
   start?: number;
 };
 
 export type ModelOutputOptions = {
+  /**
+   * Will include `profilePictureUrl` in returned models if present.
+   */
   profilePicture?: boolean;
+  /**
+   * Will include confidential fields in returned models if permitted.
+   */
   protectedFields?: boolean;
+  /**
+   * Will include related models in returned models.
+   * `singleModel` will return only models with a singleModel relation.
+   * `all` will return all types of relations.
+   *
+   * Depth is set by `relationsLevel`.
+   */
   relationModels?: string;
   relationNames?: string[];
   relationTemplateIds?: Id[];
+  /**
+   * Depth of related models to retrieve.
+   *
+   * Example:
+   * ```ts
+   * // Output model with relationsLevel === 2
+   * {
+   *   // ...
+   *   relationModels: {
+   *     accident: {
+   *       // ...
+   *       relationModels: {
+   *         employee: {
+   *           // ...
+   *         }
+   *       }
+   *     }
+   *   }
+   * }
+   * ```
+   */
   relationsLevel?: number;
+  /**
+   * Array of names of computations with `computeOn === 'read'` to skip computation.
```

Comment:
`restrictReadComputations` defines `computeOn === 'read'` computations that will have to be computed, others will be skipped.